### DISCUSSION TITLE
Inject custom header http request using QCefView browser instance in WINDOWS

### DISCUSSION THREAD

**Discussion by imofficialvivek** [CefView/QCefView#422] [Q&A]
I am using cef_binary_113.3.1+g525fa10+chromium-113.0.5672.128_windows64
And want to pass custom header via QCefVIew as done similarly in Cef using OnBeforeResourceLoad
How can i achieve this as QCefVIew takes two args as url and QCefSetting setting.?

**Comment by L-Super** [comment]
refer to：https://cefview.github.io/QCefView/docs/reference/QCefConfig#class_q_cef_config_1a60009aad390599eb5857182a32de7f23

**Reply by goodstarts** [reply]
> refer to：https://cefview.github.io/QCefView/docs/reference/QCefConfig#class_q_cef_config_1a60009aad390599eb5857182a32de7f23

<img width="1061" height="169" alt="image" src="https://github.com/user-attachments/assets/3fbb0dcf-fdf9-448e-8d3f-646bcbaf629b" />
您好，想请教下您一个问题，在利用addlocalfolderResource首次加载本地资源的时候会出现偶发崩溃，qcefview版本为1.2.1，cef内核版本为127

**Reply by L-Super** [reply]
> > refer to：https://cefview.github.io/QCefView/docs/reference/QCefConfig#class_q_cef_config_1a60009aad390599eb5857182a32de7f23
> 
> <img alt="image" width="1061" height="169" src="https://private-user-images.githubusercontent.com/128124725/475324876-3fbb0dcf-fdf9-448e-8d3f-646bcbaf629b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTQ1MzQ4NTksIm5iZiI6MTc1NDUzNDU1OSwicGF0aCI6Ii8xMjgxMjQ3MjUvNDc1MzI0ODc2LTNmYmIwZGNmLWZkZjktNDQ4ZS04ZDNmLTY0NmJjYmFmNjI5Yi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODA3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgwN1QwMjQyMzlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lNzMwZmU2MzViNGM0ZjMyZTA4ZDk5YTRhNzg5ZThmMDZlNjgyNzQwZGExNTZlMGI3MDFiNGUxZTJhMjgzZTU4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.k63a8yFW_RZEnazj7H7zImIQU26iRUFnr4C8F_1-VxM"> 您好，想请教下您一个问题，在利用addlocalfolderResource首次加载本地资源的时候会出现偶发崩溃，qcefview版本为1.2.1，cef内核版本为127

可以提一个issue

**Reply by goodstarts** [reply]
好的，谢谢
